missions/2PzD_Template_v3_0.VR/customization/geardefs/Generic/gearDefGEN.sqf

```diff
@@ -110,7 +110,7 @@
 #define GEN_BandagePacking      "ACE_packingBandage"
 #define GEN_BandageQClot        "ACE_quikclot"
 #define GEN_TQ                  "ACE_tourniquet"
-#define GEN_Pain                "ACE_painkillers"
+#define GEN_Pain                "ACE_Banana"
```

Comment:
I think we should just remove the GEN_Pain everywhere, as funny as it would be to see the faces of our medics when they spawn with 10 bananas :D

Reply:
GEN_Pain has been removed in the latest commit